dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -891,6 +930,8 @@ private String buildSql( EventSearchParams params, List<OrganisationUnit> organi
 
         sqlBuilder.append( ") as cm on event.psi_id=cm.psic_id " );
 
+        sqlBuilder.append( RELATIONSHIP_IDS_QUERY );
```

Comment:
Is it possible to do if (!skipRelationships) here? Or would it mess up the query?